shared/haml/hamburger.haml

```diff
@@ -82,7 +103,7 @@
   end
 
   # Do we show hamburger on all widths, only mobile, or not at all?
-  if show_teacher_options == "show-always" || show_student_options == "show-always" || show_pegasus_options == "show-always"
+  if show_teacher_options == "show-always" || show_student_options == "show-always" || show_pegasus_options == "show-always" || language != "en"
```

Comment:
I would prefer that we key off of another `_show_` variable rather than introducing language into this condition.